Libs/LibOpenRaid/GetPlayerInformation.lua

```diff
@@ -818,7 +821,9 @@ function openRaidLib.CooldownManager.GetPlayerCooldownStatus(spellId)
                 return ceil(timeLeft), chargesAvailable, startTimeOffset, duration, buffDuration
             end
         else
-            local start, duration = GetSpellCooldown(spellId)
+            local spellCooldownInfo = GetSpellCooldown(spellId)
+            local start = spellCooldownInfo["startTime"]
+            local duration = spellCooldownInfo["duration"]
```

Comment:
I don't like the usage of indexing by string keys. Can this be converted to dot notation?

Reply:
Sure, I've updated the code